.github/workflows/build_linux_packages.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           amdgpu_families: ${{ inputs.amdgpu_families }}
           package_version: ${{ inputs.package_version }}
           extra_cmake_options: ${{ inputs.extra_cmake_options }}
-          build_dir: build
+          BUILD_DIR_BASH: build
```

Comment:
I'd prefer to just call this `build_dir` (or `BUILD_DIR`). We still need to remove the non-bash path handling around here: https://github.com/ROCm/TheRock/blob/a2dd40ae7de646eb1d90edc27971de918280a9f6/.github/workflows/build_windows_packages.yml#L42-L62

Could clean that up together with changes to the other windows workflows